recipes/archicad-apidevkit/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+cmake_minimum_required(VERSION 3.15)
+
+project(test_package CXX)
+
+function(SetCompilerOptions target ac_version)
+    if(ac_version GREATER "26")
+        target_compile_features("${target}" PUBLIC cxx_std_17)
+    else()
+        target_compile_features("${target}" PUBLIC cxx_std_14)
+    endif()
+    target_compile_definitions("${target}" PUBLIC "\$<\$<CONFIG:Debug>:DEBUG>")
+    if(WIN32)
+        target_compile_options(
+            "${target}" PRIVATE
+            /W4 /WX
+            /Zc:wchar_t-
+            /wd4499
+            /EHsc
+        )
+    else()
+        target_compile_options(
+            "${target}" PRIVATE
+            -Wall -Wextra -Werror
+            -fvisibility=hidden
+            -Wno-multichar
+            -Wno-ctor-dtor-privacy
+            -Wno-invalid-offsetof
+            -Wno-ignored-qualifiers
+            -Wno-reorder
+            -Wno-overloaded-virtual
+            -Wno-unused-parameter
+            -Wno-deprecated
+        )
+    endif()
+endfunction()
+
+set(AC_API_DEVKIT_DIR "$ENV{AC_API_DEVKIT_DIR}" CACHE PATH "API DevKit directory.")
+
+find_package(archicad-apidevkit REQUIRED CONFIG)
+
+add_executable(test_package test_gsroot.cpp)
```

Comment:
Is there a particular reason to not build or run this executable? 

It's the only unusual part of the recipe

Reply:
In the latest version everything seems to be OK.

Reply:
The DLLs are not distributed, they are shipped with Archicad. The SDK only provides import libraries on Windows for example.

You would get missing DLL error if you tried to run the built executable without Archicad.